backend/src/main/java/org/sefglobal/core/academix/model/Category.java

```diff
@@ -23,6 +24,7 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @NotNull
     @Column(columnDefinition = "TEXT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci")
```

Comment:
```suggestion
    @Column(columnDefinition = "TEXT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci", nullable = false)
```
@Piumal1999 you can use the already existing `@Column` instead of adding a new annotation

Reply:
Updated. Thanks @YohanAvishke

Reply:
Nice job